jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/python/PythonLambdaHandlerResolverTest.kt

```diff
@@ -140,6 +141,20 @@ class PythonLambdaHandlerResolverTest {
         assertHandler("app.handle", true)
     }
 
+    @Test
+    fun pyTestHandlersIgnored() {
+        createPyTestHandler("hello_world/app.py")
+
+        assertHandler("doesnt_exist", false)
```

Comment:
This should be `assertHandler("hello_world/app.test_handle", false)`

Reply:
Fixed